fwb/app/(auth)/(routes)/fre3/page.tsx

```diff
@@ -93,10 +95,6 @@ export default function UserFlowPage3() {
     window.open("https://www.instagram.com/");
   };
 
-  const handlefacebook = () => {
```

Comment:
Functions could be camel case? makes it easily readable. e.g handleFacebook

Reply:
Updated the naming.